### PR TITLE
Release 7.0.1

### PR DESCRIPTION
**Pull request by Carreau**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bollwyvl** [comment]
So close! Having a look at the windows builds...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,13 +18,13 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3
```
Comment (bollwyvl) [inline recipe/meta.yaml]: ah. take these selectors off, as the 
```
skip: True  # [py2k]
``` 
is sufficient. And will keep the windows paths from going :monkey:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -18,13 +18,13 @@ build:
...
     - pip
   run:
-    - python
+    - python >=3
```
Comment (bollwyvl) [inline recipe/meta.yaml]: also this

File: recipe/meta.yaml [hunk 3]
```diff
@@ -78,6 +78,8 @@ extra:
   recipe-maintainers:
     - Carreau
     - jakirkham
+    - fperez
+    - ellisonbg
```
Comment (bollwyvl) [inline recipe/meta.yaml]: and feel free to add me if so inclined...

**Comment by blink1073** [comment]
Closing in favor of #58

**Comment by blink1073** [comment]
Ha, he says without having commit rights...

**Comment by Carreau** [comment]
> Closing in favor of #58

Closed, Apologies I'll be a bit busy today. THanks for taking over.

**Pull request closed without merging**